### PR TITLE
[BULK] - DocuTune - Rebranding of Azure Active Directory to Microsoft Entra terminology

### PR DESCRIPTION
**Pull request by alexbuckgit**
Applying Azure AD rebranding changes as announced on July 11, 2023 and outlined in https://aka.ms/AzureADNewName. Changes are part of the Microsoft-wide rebranding effort.

CorrelationId: 2520d58e-25ff-469e-af14-2834d2a1505e
PointOfContact: @CelesteDG

#docutune

### CODE REVIEW

**Comment by alexbuckgit** [comment]
#docutune-draft

**Comment by CelesteDG** [comment]
@msewaweru Please review. Let's chat as I think we may need to revert some of the references to Azure AD that got changed especially when it's referring to the modules all up.

File: microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md [hunk 1]
```diff
@@ -75,7 +77,7 @@ The following is the categorization of the Azure AD cmdlets. Select a category t
 
 ### Application proxy connector management
 
-|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
+|Microsoft Entra cmdlets | Microsoft Graph PowerShell cmdlets|
```
Comment (CelesteDG) [inline microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md:80]:
```suggestion
|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
```

File: microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md [hunk 2]
```diff
@@ -89,7 +91,7 @@ The following is the categorization of the Azure AD cmdlets. Select a category t
 
 ### Applications
 
-|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
+|Microsoft Entra cmdlets | Microsoft Graph PowerShell cmdlets|
```
Comment (CelesteDG) [inline microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md:94]:
```suggestion
|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
```

File: microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md [hunk 3]
```diff
@@ -112,9 +114,11 @@ The following is the categorization of the Azure AD cmdlets. Select a category t
 |Set-AzureADApplication|[Update-MgApplication](/powershell/module/microsoft.graph.applications/update-mgapplication)|
 |Set-AzureADApplicationLogo|[Set-MgApplicationLogo](/powershell/module/microsoft.graph.applications/set-mgapplicationlogo)|
 
-### Azure AD
+<a name='azure-ad'></a>
+
+### Microsoft Entra ID
 
-|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
+|Microsoft Entra cmdlets | Microsoft Graph PowerShell cmdlets|
```
Comment (CelesteDG) [inline microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md:121]:
```suggestion
|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
```

File: microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md [hunk 4]
```diff
@@ -172,7 +176,7 @@ The following is the categorization of the Azure AD cmdlets. Select a category t
 
 ### Certificate Authorities
 
-|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
+|Microsoft Entra cmdlets | Microsoft Graph PowerShell cmdlets|
```
Comment (CelesteDG) [inline microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md:179]:
```suggestion
|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
```

File: microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md [hunk 5]
```diff
@@ -181,14 +185,14 @@ The following is the categorization of the Azure AD cmdlets. Select a category t
 
 ### Connect to your directory
 
-|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
+|Microsoft Entra cmdlets | Microsoft Graph PowerShell cmdlets|
```
Comment (CelesteDG) [inline microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md:188]:
```suggestion
|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
```

File: microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md [hunk 6]
```diff
@@ -181,14 +185,14 @@ The following is the categorization of the Azure AD cmdlets. Select a category t
...
 |------------|--------------|
 |Connect-AzureAD|[Connect-MgGraph](authentication-commands.md#using-connect-mggraph)|
 |Disconnect-AzureAD|[Disconnect-MgGraph](authentication-commands.md#using-disconnect-mggraph)|
 
 ### Contacts
 
-|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
+|Microsoft Entra cmdlets | Microsoft Graph PowerShell cmdlets|
```
Comment (CelesteDG) [inline microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md:195]:
```suggestion
|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
```

File: microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md [hunk 7]
```diff
@@ -200,19 +204,19 @@ The following is the categorization of the Azure AD cmdlets. Select a category t
 
 ### Contracts
 
-|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
+|Microsoft Entra cmdlets | Microsoft Graph PowerShell cmdlets|
```
Comment (CelesteDG) [inline microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md:207]:
```suggestion
|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
```

File: microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md [hunk 8]
```diff
@@ -200,19 +204,19 @@ The following is the categorization of the Azure AD cmdlets. Select a category t
...
 |------------|--------------|
 |Get-AzureADContract|[Get-MgContract](/powershell/module/microsoft.graph.identity.directorymanagement/get-mgcontract)|
 
 ### Deleted objects
 
-|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
+|Microsoft Entra cmdlets | Microsoft Graph PowerShell cmdlets|
```
Comment (CelesteDG) [inline microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md:213]:
```suggestion
|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
```

File: microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md [hunk 9]
```diff
@@ -200,19 +204,19 @@ The following is the categorization of the Azure AD cmdlets. Select a category t
...
 |------------|--------------|
 |Restore-AzureADDeletedApplication|[Restore-MgDirectoryDeletedItem](/powershell/module/microsoft.graph.identity.directorymanagement/restore-mgdirectorydeleteditem)|
 
 ### Devices
 
-|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
+|Microsoft Entra cmdlets | Microsoft Graph PowerShell cmdlets|
```
Comment (CelesteDG) [inline microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md:219]:
```suggestion
|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
```

File: microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md [hunk 10]
```diff
@@ -228,21 +232,21 @@ The following is the categorization of the Azure AD cmdlets. Select a category t
 
 ### Directory
 
-|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
+|Microsoft Entra cmdlets | Microsoft Graph PowerShell cmdlets|
```
Comment (CelesteDG) [inline microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md:235]:
```suggestion
|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
```

File: microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md [hunk 11]
```diff
@@ -228,21 +232,21 @@ The following is the categorization of the Azure AD cmdlets. Select a category t
...
 |------------|--------------|
 |Get-AzureADSubscribedSku|[Get-MgSubscribedSku](/powershell/module/microsoft.graph.identity.directorymanagement/get-mgsubscribedsku)|
 |Get-AzureADTenantDetail|[Get-MgOrganization](/powershell/module/microsoft.graph.identity.directorymanagement/get-mgorganization)|
 |Set-AzureADTenantDetail|[Update-MgOrganization](/powershell/module/microsoft.graph.identity.directorymanagement/update-mgorganization)|
 
 ### Directory Objects
 
-|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
+|Microsoft Entra cmdlets | Microsoft Graph PowerShell cmdlets|
```
Comment (CelesteDG) [inline microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md:243]:
```suggestion
|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
```

File: microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md [hunk 12]
```diff
@@ -228,21 +232,21 @@ The following is the categorization of the Azure AD cmdlets. Select a category t
...
 |------------|--------------|
 |Get-AzureADObjectByObjectId|[Get-MgDirectoryObjectById](/powershell/module/microsoft.graph.directoryobjects/get-mgdirectoryobjectbyid)|
 
 ### Directory roles
 
-|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
+|Microsoft Entra cmdlets | Microsoft Graph PowerShell cmdlets|
```
Comment (CelesteDG) [inline microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md:249]:
```suggestion
|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
```

File: microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md [hunk 13]
```diff
@@ -260,7 +264,7 @@ The following is the categorization of the Azure AD cmdlets. Select a category t
 
 ### Domains
 
-|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
+|Microsoft Entra cmdlets | Microsoft Graph PowerShell cmdlets|
```
Comment (CelesteDG) [inline microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md:267]:
```suggestion
|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
```

File: microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md [hunk 14]
```diff
@@ -273,13 +277,13 @@ The following is the categorization of the Azure AD cmdlets. Select a category t
 
 ### Extension properties
 
-|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
+|Microsoft Entra cmdlets | Microsoft Graph PowerShell cmdlets|
```
Comment (CelesteDG) [inline microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md:280]:
```suggestion
|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
```

File: microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md [hunk 15]
```diff
@@ -273,13 +277,13 @@ The following is the categorization of the Azure AD cmdlets. Select a category t
...
 |------------|--------------|
 |Get-AzureADExtensionProperty|[Get-MgDirectoryObjectAvailableExtensionProperty](/powershell/module/microsoft.graph.directoryobjects/get-mgdirectoryobjectavailableextensionproperty)|
 
 ### Groups
 
-|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
+|Microsoft Entra cmdlets | Microsoft Graph PowerShell cmdlets|
```
Comment (CelesteDG) [inline microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md:286]:
```suggestion
|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
```

File: microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md [hunk 16]
```diff
@@ -310,21 +314,21 @@ The following is the categorization of the Azure AD cmdlets. Select a category t
 
 ### Oauth2
 
-|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
+|Microsoft Entra cmdlets | Microsoft Graph PowerShell cmdlets|
```
Comment (CelesteDG) [inline microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md:317]:
```suggestion
|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
```

File: microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md [hunk 17]
```diff
@@ -310,21 +314,21 @@ The following is the categorization of the Azure AD cmdlets. Select a category t
...
 |------------|--------------|
 |Get-AzureADOAuth2PermissionGrant|[Get-MgOauth2PermissionGrant](/powershell/module/microsoft.graph.identity.signins/get-mgoauth2permissiongrant)|
 |Remove-AzureADOAuth2PermissionGrant|[Remove-MgOauth2PermissionGrant](/powershell/module/microsoft.graph.identity.signins/remove-mgoauth2permissiongrant)|
 
 ### Policies
 
-|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
+|Microsoft Entra cmdlets | Microsoft Graph PowerShell cmdlets|
```
Comment (CelesteDG) [inline microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md:324]:
```suggestion
|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
```

File: microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md [hunk 18]
```diff
@@ -310,21 +314,21 @@ The following is the categorization of the Azure AD cmdlets. Select a category t
...
 |------------|--------------|
 |Get-AzureADMSAuthorizationPolicy|[Get-MgPolicyAuthorizationPolicy](/powershell/module/microsoft.graph.identity.signins/get-mgpolicyauthorizationpolicy)|
 |Set-AzureADMSAuthorizationPolicy|[Update-MgPolicyAuthorizationPolicy](/powershell/module/microsoft.graph.identity.signins/update-mgpolicyauthorizationpolicy)|
 
 ### Service principals
 
-|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
+|Microsoft Entra cmdlets | Microsoft Graph PowerShell cmdlets|
```
Comment (CelesteDG) [inline microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md:331]:
```suggestion
|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
```

File: microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md [hunk 19]
```diff
@@ -352,7 +356,7 @@ The following is the categorization of the Azure AD cmdlets. Select a category t
 
 ### Users
 
-|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
+|Microsoft Entra cmdlets | Microsoft Graph PowerShell cmdlets|
```
Comment (CelesteDG) [inline microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md:359]:
```suggestion
|Azure AD cmdlets | Microsoft Graph PowerShell cmdlets|
```

File: microsoftgraph/docs-conceptual/migration-steps.md [hunk 20]
```diff
@@ -12,19 +12,19 @@ ms.reviewer: maisarissi, ianfarr
 
 # Upgrade from Azure AD PowerShell to Microsoft Graph PowerShell
 
-Azure AD, Azure AD Preview and MSOnline PowerShell modules are planned for deprecation. Microsoft Graph PowerShell is the PowerShell module to use for interacting with Azure AD and other Microsoft services.
+Microsoft Entra ID, Microsoft Entra ID Preview and MSOnline PowerShell modules are planned for deprecation. Microsoft Graph PowerShell is the PowerShell module to use for interacting with Microsoft Entra ID and other Microsoft services.
```
Comment (CelesteDG) [inline microsoftgraph/docs-conceptual/migration-steps.md:15]:
```suggestion
Azure AD, Azure AD Preview and MSOnline PowerShell modules are planned for deprecation. Microsoft Graph PowerShell is the PowerShell module to use for interacting with Microsoft Entra ID and other Microsoft services.
```

**Review by CelesteDG — CHANGES_REQUESTED** [review]

**Review by CelesteDG — APPROVED** [review]

**Comment by msewaweru** [comment]
Thanks, @CelesteDG for taking up the review work of this PR while I was away. Much appreciated.

**Pull request merged**